CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 [![PyPI version](https://badge.fury.io/py/iotedgehubdev.svg)](https://badge.fury.io/py/iotedgehubdev)
+## 0.8.0 - 2019-04-01
+### Added
+* Add module twin support
+* Support Docker daemon with ssh:// URL for Python 2
```

Comment:
Its a fix instead of new added feature. How about update the Changed with "Upgrade docker-py dependency to support connect remote Docker engine with ssh://"

Reply:
OK